API/resources/etc/tester.py

```diff
@@ -219,6 +229,10 @@ def parse_arguments():
     parser.add_argument('--testfile', metavar='file',
                         help='Test file with path')
 
+    parser.add_argument('--coverage-port',
+                        metavar='PORT',
+                        help='Specify the PORT for jerry-debugger to calculate the JS source code coverage')
```

Comment:
Do we really need this option? In the [previos patch (ARTIK053 target)](https://github.com/Samsung/js-remote-test/blob/3e01645c23b65bd941113f4a6abeae4749c30911/driver.py#L69) it was enough to have a `--coverage` option (HOST:PORT).

Reply:
Yes, but but this is just another simplification. It will runs on the server side, where the port definition only is sufficient. There is no `--coverage` option in this case.

Reply:
You are right.